clients/csharp-uwp/UWPVoiceAssistantSample/DirectLineSpeechDialogBackend.cs

```diff
@@ -92,6 +99,8 @@ public DirectLineSpeechDialogBackend()
         /// </summary>
         public object ConfirmationModel { get; set; }
 
+        private string keywordFilePath;
+
         /// <summary>
         /// Gets or sets the configuration object that will be used when creating the

```

Comment:
I wonder if we should change the name of this (on the overall backend) to reflect that it's not really "initializing" anymore, but rather "making sure that it's in a good state." Another one of those "can think about it later" things, just want to call it out!

Reply:
If you are talking about the method name "InitializeAsync", the only reason I'd be against changing it is that it is not meant to be DLS specific. This is the DialogBackend method that is called to initialize the class before a conversation is started. It just so happens that for DLS, initialization means checking for config updates and recreating the connector if necessary.